src/liballoc/collections/btree/map.rs

```diff
@@ -490,6 +490,8 @@ struct MergeIter<K, V, I: Iterator<Item = (K, V)>> {
 impl<K: Ord, V> BTreeMap<K, V> {
     /// Makes a new empty BTreeMap with a reasonable choice for B.
```

Comment:
While we're in here, I think the "with a reasonable choice for B" bit should be removed, since that implies that there's some other constructor that lets you choose a B.

Reply:
will get this change made in another pr, don't want this one stalling because of it :)